addOns/authhelper/src/test/java/org/zaproxy/addon/authhelper/report/ExtensionAuthhelperReportUnitTest.java

```diff
@@ -197,6 +200,47 @@ void shouldGenerateFilledAuthJsonReport() throws Exception {
         assertThat(statistics.getJSONObject(2).getInt("value"), is(equalTo(5678)));
     }
 
+    @Test
+    void shouldGenerateFilledAuthJsonReportHandlingSpecialCharacters() throws Exception {
+        // Given
+        ExtensionReports extRep = new ExtensionReports();
+        String templateName = "auth-report-json";
+        Template template = getTemplateFromYamlFile(templateName);
+        File f = File.createTempFile(templateName, template.getExtension());
+        ReportData reportData = getGenericReportData(templateName);
+        reportData.setSections(template.getSections());
+        AuthReportData ard = new AuthReportData();
+        reportData.addReportObjects("authdata", ard);
+
+        ard.setSite("https://www.example.com");
+        String afEnv =
+                """
+                  env:
+                  contexts:
+                      name: 'some "quote" name'
+                """;
+        ard.setAfEnv(afEnv);
+        ard.addSummaryItem(true, "summary.1", "Bob's \"Item\"");
+        ard.addSummaryItem(true, "summary.\"2\"", "Foo bar");
+        ard.addStatsItem("stats.auth.1", "foo \"random\" bar", 123);
+        ard.addStatsItem("stats.foo.oops \"foo\" bar", "global", 0);
+        // When
+        File r = extRep.generateReport(reportData, template, f.getAbsolutePath(), false);
+        String report = new String(Files.readAllBytes(r.toPath()));
+
+        // Then
+        LocalDateTime localDateTime = LocalDateTime.now();
+        ZonedDateTime zonedDateTime = localDateTime.atZone(ZoneId.systemDefault());
+        String current = zonedDateTime.format(DateTimeFormatter.RFC_1123_DATE_TIME);
+        String expected =
+                "{\n\t\"@programName\": \"ZAP\",\n\t\"@version\": \"Test Build\",\n\t\"@generated\": \"@@@replace@@@\",\n\t\"site\":  \"https:\\/\\/www.example.com\"\n\t\n\t,\"summaryItems\": [\n\t\t{\n\t\t\t\"description\": \"Bob's \\\"Item\\\"\",\n\t\t\t\"passed\": true,\n\t\t\t\"key\": \"summary.1\"\n\t\t},\n\t\t{\n\t\t\t\"description\": \"Foo bar\",\n\t\t\t\"passed\": true,\n\t\t\t\"key\": \"summary.\\\"2\\\"\"\n\t\t}\n\t]\n\t\n\t\n\t,\"afEnv\": \"  env:\\n  contexts:\\n      name: 'some \\\"quote\\\" name'\\n\"\n\t\n\t\n\t,\"statistics\": [\n\t\t{\n\t\t\t\"key\": \"stats.auth.1\",\n\t\t\t\"scope\": \"foo \\\"random\\\" bar\",\n\t\t\t\"value\": 123\n\t\t},\n\t\t{\n\t\t\t\"key\": \"stats.foo.oops \\\"foo\\\" bar\",\n\t\t\t\"scope\": \"global\",\n\t\t\t\"value\": 0\n\t\t}\n\t]\n\t\n}\n"
```

Comment:
Would still be good to use text block to improve the readability.

Reply:
I'll see what I can do.

I think I actually got it in this form from the tests or something...

Reply:
The escapes should be dropped.

Reply:
You mean the tabs?

Reply:
Or really all of them?